stand_with_ukraine.js

```diff
@@ -1,7 +1,7 @@
+/* global swu_options */
 document.addEventListener('DOMContentLoaded', function() {
-	let body = document.getElementsByTagName('body')[0];
-	let div  = document.createElement('div');
+	const div  = document.createElement('div');
 	div.id = 'stand_with_ukraine_overlay';
 	div.innerHTML = '<a href="' + swu_options.url + '">' + swu_options.text + '</a>';
-	document.body.insertBefore(div, body.firstChild);
+	document.body.prepend(div);
```

Comment:
`prepend` seems to not be supported in IE. So if we're aiming for support of older browsers this seems like it breaks support? Have I understood that right?

What does this change fix? Was something broken before?

I'm not seeing any errors.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend
If we supported the old IE world, we should do not use `prepend` and leave the first idea.

Reply:
Another option could be:
`document.body.insertAdjacentElement('beforebegin', div);`
that works with IE > 5

Reply:
you can try this test (also in the browser console of this page) which is much faster "insertAdjacentElement" just because the method argument is targeting the "right" element
```
const swu_options = {url:'',text:''}
function insertAE() {
  const div  = document.createElement('div');
  div.id = 'stand_with_ukraine_overlay';
  div.innerHTML = '<a href="' + swu_options.url + '">' + swu_options.text + '</a>';
  document.body.insertAdjacentElement('beforebegin', div);
}

function insertBE() {
  const body = document.getElementsByTagName('body')[0];
  const div  = document.createElement('div');
  div.id = 'stand_with_ukraine_overlay';
  div.innerHTML = '<a title="' + swu_options.text + '" href="' + swu_options.url + '">' + swu_options.hashtag + '</a>';
  document.body.insertBefore(div, body.firstChild);
}

performance.mark('mark-1')
Array.from(Array(5), insertBE)
Array.from(Array(5), insertAE)

performance.mark('mark-2')
Array.from(Array(1000), insertBE)

performance.mark('mark-3')
Array.from(Array(1000), insertAE)

performance.mark('mark-4')

// store test measurement
performance.measure('insertBefore', 'mark-2', 'mark-3' )
performance.measure('insertAdjacentElement2', 'mark-3', 'mark-4' )

// log results
console.log("insertBefore", performance.getEntriesByName('insertBefore')[0].duration)
console.log("insertAdjacentElement", performance.getEntriesByName('insertAdjacentElement2')[0].duration)
console.log(performance.getEntries())
```